bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BFutureType.java

```diff
@@ -81,4 +81,15 @@ public boolean equals(Object obj) {
         return TypeChecker.checkIsType(constraint, other.constraint);
     }
 
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(getConstraintString());
+        sb.insert(0, TypeConstants.FUTURE_TNAME);
```

Comment:
Shall we not use a `StringBuilder` here because it doesn't add any value as we are using `insert` function. Shall we use simple String concatenation?

Reply:
Addressed with the commit 6164d67